examples/huggingface/neural_speed/README.md

```diff
@@ -41,33 +41,47 @@ pip install -r requirements.txt
 
 ## 1. Performance
 
+### INT4 Inference
 ``` bash
-# int4 with group-size=128
-# --not_quant use fp32 inference
-OMP_NUM_THREADS=<physical cores num> numactl -m <node N> -C <cpu list> python runtime_example.py \
+# int4 with group-size=32
+OMP_NUM_THREADS=<physical cores num> numactl -m <node N> -C <cpu list> python run_inference.py \
     --model_path ./Llama2 \
     --prompt "Once upon a time, there existed a little girl," \
     --max_new_tokens 32 \
     --group_size 128
 ```
 
+### FP32 Inference
+
+``` bash
+OMP_NUM_THREADS=<physical cores num> numactl -m <node N> -C <cpu list> python run_inference.py \
+    --model_path ./Llama2-fp32 \
+    --prompt "Once upon a time, there existed a little girl," \
+    --max_new_tokens 32 \
+    --not_quant
+```
+
+
 ## 2. Accuracy
 
+> Note: If you are trying models generated by [AutoRound](../pytorch/text-generation/quantization/auto_round/README.md), please use script `run_autoround_example.py` for accuracy evaluation.
+
+
+### INT4 Accuracy
+
 ```bash
 # int4 with group-size=32
-python runtime_acc.py \
+python run_accuracy.py \
     --model_name ./Llama2 \
     --tasks "lambada_openai"
 ```
 
+### FP32 Accuracy
 
-> Note: If you are trying models generated by [autoround](../pytorch/text-generation/quantization/), need to disable `model_format` and `use_gptq` these two arguments.
-> ```bash
->   # parser.add_argument('--model_format', type=str, default="runtime")
->   # parser.add_argument('--use_gptq', action='store_true')
->   results = evaluate(
->        model="hf-causal",
->        model_args=f'pretrained="{args.model_name}", dtype=float32',
->        tasks=[f"{args.tasks}"]
->   )
-> ```
+```bash
+# int4 with group-size=32
+python run_accuracy.py \
+    --model_name ./Llama2-fp32 \
+    --tasks "lambada_openai" \
+    --model_format "torch"
```

Comment:
Now,  there's no need for ‘runtime’ format ?

Reply:
To run fp32 accuracy, runtime do not support